src/opera/instance/node.py

```diff
@@ -11,6 +13,7 @@ def __init__(self, template, instance_id):
 
         self.in_edges = {}  # This gets filled by other instances for us.
         self.out_edges = {}  # This is what we fill during the linking phase.
+        self.notified = False  # This indicates whether the node has been notified.
```

Comment:
No, this is only used for node executor within notify function to indicate which node has already been notified so that the process can stop eventually. This also doesn't prevent calling notify multiple times on the same nodes.